tests/test_sca_models.py

```diff
@@ -33,6 +33,14 @@ class TestModel:
                             "path": "Pipfile",
                         }
                     ],
+                    "ignore_fix_available": True,
```

Comment:
typo line 29: `minimum_severity` I don't know if it was on purpose

Reply:
which makes me think that the `test_schema_loads` is irrelevant, the schemas can take whatever data

Reply:
Good catch, that's an old typo !

Reply:
Not really whatever, if the field is specified the test will try to evaluate it to instantiate the class. So at least there is some type evaluation. In the previous case, there is a default value for `minimum_severity` so we missed the typo.